pom.xml

```diff
@@ -199,6 +201,11 @@
 				<artifactId>jdom2</artifactId>
 				<version>2.0.6.1</version>
 			</dependency>
+			<dependency>
```

Comment:
Ik zie dat je een paar dependencies overhevelt naar de parent pom; zaten er in de pom van core niet ook nog een paar in een dependencyManagement blok daar?
Misschien beter om gelijk alles te doen en naar dependencyManagement van de parent-pom te zetten?

Reply:
Goed ik wist niet meer dat die al in 7.8 zaten.

Reply:
Geen probleem. Dezelfde gedachte.